### PR TITLE
Remove the lint about non nullable binding instances

### PR DESCRIPTION
**Pull request by ValentinVignal**
Fixes https://github.com/creativecreatorormaybenot/performance/issues/18

I used the fix recommended by the flutter team: https://docs.flutter.dev/development/tools/sdk/release-notes/release-notes-3.0.0#your-code

I also run `flutter pub get`, tell me if you want me to remove those changes

### CODE REVIEW

**Review by creativecreatorormaybenot — CHANGES_REQUESTED** [review]
[`SchedulerBinding.instance`](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/instance.html) is no longer nullable on stable, which means that the `_ambiguate` function is unnecessary.

**Comment by ValentinVignal** [comment]
I can not use `_ambiguate` if you want but it will be a breaking change. A project that didn't upgrade to flutter 3 won't be able to use the new version of this package. Is that fine ?

**Comment by creativecreatorormaybenot** [comment]
@ValentinVignal It is not a breaking change since we expect everyone using the package to be on the stable channel of Flutter (this is also what the Pub analysis tool uses).

**Comment by ValentinVignal** [comment]
@creativecreatorormaybenot fair enough, I removed `_ambiguate` in https://github.com/creativecreatorormaybenot/performance/pull/19/commits/c57bbd97379bad6cfdc93c8a92a030e766c89a5b

**Review by creativecreatorormaybenot — APPROVED** [review]
lgtm

Two additional things I would change:
1. Could you bump the version and add an entry to the CHANGELOG.md file? That would be version `0.1.1`.
2. I realize this is my mistake, but the `performance/pubspec.lock` should not be committed (only the lock file of the example app). Could you delete that and add it to `performance/.gitignore`?

Let me know if you want to do that as part of this PR. Otherwise, I will just create a follow-up PR.

**Comment by ValentinVignal** [comment]
@creativecreatorormaybenot That is fine, I can do it in this PR. I pushed the changes, tell me if something else needs to be changed.

File: performance/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 0.1.1
+
+* Migrate the code to Flutter 3 by considering `SchedulerBinding.instance` non-nullable.
```
Comment (creativecreatorormaybenot) [inline performance/CHANGELOG.md:3]:
```suggestion
* Fixed warnings introduced by Flutter 3 (`SchedulerBinding.instance` is now non-nullable).
```

File: performance/pubspec.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 description: Performance overlay package for Flutter apps that works on web.
-version: 0.1.0+1
+version: 0.1.1
 homepage: https://github.com/creativecreatorormaybenot/performance/tree/main/performance
 
 environment:
-  sdk: '>=2.14.0 <3.0.0'
-  flutter: '>=1.17.0'
+  sdk: ">=2.14.0 <3.0.0"
+  flutter: ">=1.17.0"
```
Comment (creativecreatorormaybenot) [inline performance/pubspec.yaml]:
```suggestion
  sdk: '>=2.14.0 <3.0.0'
  flutter: '>=1.17.0'
```

**Review by creativecreatorormaybenot — APPROVED** [review]
LGTM

**Comment by creativecreatorormaybenot** [comment]
@ValentinVignal Thank you so much for your contribution!

**Comment by ValentinVignal** [comment]
@creativecreatorormaybenot Is there something else I need to do for it to be merged and released?

**Comment by creativecreatorormaybenot** [comment]
@ValentinVignal your commits were not signed. But I removed this requirement now as I am going to squash anyway (in which case GitHub will sign the squash commit).

**Pull request merged**